engine/requirements.in

```diff
@@ -9,7 +9,6 @@ django-cors-headers==3.7.0
 # pyroscope-io==0.8.1
 django-dbconn-retry==0.1.7
 django-debug-toolbar==4.1
-django-deprecate-fields==0.1.1
```

Comment:
this is a random change, but we discussed it [here](https://raintank-corp.slack.com/archives/C081TNWM73N/p1732190163750779?thread_ts=1732135139.226719&cid=C081TNWM73N) (since it's only a one-liner change + this library isn't used, decided to just squeeze it in this PR)